content/_code-samples/secure-signing/py/sign-payment.py

```diff
@@ -1,32 +1,20 @@
-from xrpl.core import keypairs 
-
-# Define network client
-from xrpl.clients import JsonRpcClient
-JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
-client = JsonRpcClient(JSON_RPC_URL)
-
 # Define signer address
+import os  
+my_secret = os.getenv("MY_SECRET")
 from xrpl.wallet import Wallet
-test_wallet_1 = Wallet(seed="shrPSF6vV3v3yoND9J3NeKks6M3xd")
-print(test_wallet_1.classic_address) # "raaFKKmgf6CRZttTVABeTcsqzRQ51bNR6Q"
-
-# Prepate the tranaaction
-from xrpl.ledger import get_latest_validated_ledger_sequence
-current_validated_ledger = get_latest_validated_ledger_sequence(client)
-min_ledger = current_validated_ledger + 1
-max_ledger = min_ledger + 20
+wallet = Wallet(seed="MY_SECRET")
+print(wallet.classic_address) # "raaFKKmgf6CRZttTVABeTcsqzRQ51bNR6Q"
 
 from xrpl.models.transactions import Payment
-# not working in testing
-# from xrpl.utils import xrp_to_drops
+from xrpl.utils import xrp_to_drops
 my_payment = Payment(
     account=test_wallet_1.classic_address,
-    amount="2200000",
+    amount=xrp_to_drops(22),
+    fee=xrp_to_drops(10),
```

Comment:
This would be a 10 XRP fee. I don't think you want that.

```suggestion
    fee="10"
```